### PR TITLE
mpls: proposed fixings after telechat

### PR DESCRIPTION
**Pull request by vb-balazs**
Proposed changes:
0, fixing Roman Danyliw <discuss> comment
1, change introduction to highlight additional functions of data plane
and the focus of the draft
2, highlight that implementation of PREOF is out of scope, pointing
to related IEEE work (i.e., 802.1CB)
3, fixing the POF section and its configuration via controller plane

### CODE REVIEW

**Review by dfedyk — APPROVED** [review]

File: mpls/draft-ietf-detnet-mpls.xml [hunk 1]
```diff
@@ -110,13 +110,21 @@
...
+   data plane to establish and support DetNet flows.
+   The DetNet Architecture models the DetNet related data plane
+   functions decomposed into two sub-layers: a service sub-layer and a
+   forwarding sub-layer.  The service sub-layer is used to provide
+   DetNet service functions such as protection and reordering.  At the
+   DetNet data plane a new set of functions (PREOF) provide the service
+   sub-layer specific tasks. The forwarding sub-layer is used to provide
+   forwarding assurance (low loss, assured latency, and limited
+   out-of-order delivery) via already existing Traffic Engineering
+   and queuing tools. The use of the functionalities of the DetNet
```
Comment (louberger) [inline mpls/draft-ietf-detnet-mpls.xml:123]: New tools (queuing mechanisms, TSpecs are also allowed", so I'd prefer to clarify or remove "via already existing Traffic Engineering  and queuing tools."

File: mpls/draft-ietf-detnet-mpls.xml [hunk 2]
```diff
@@ -652,6 +660,11 @@ DFx = DetNet member flow x over a TE LSP
      encapsulations for consistency but there is no hard requirement in
      this regard.
    </t>
+   <t>
+     Implementation details of PREOF functions is out of scope for this document.
+     <xref target="IEEE802.1CB-2017"/> specifies replication and elimination
```
Comment (louberger) [inline mpls/draft-ietf-detnet-mpls.xml:665]: s/specifies/defines an equivalent

File: mpls/draft-ietf-detnet-mpls.xml [hunk 3]
```diff
@@ -732,6 +745,23 @@ DFx = DetNet member flow x over a TE LSP
...
+     <t>
+	   The latency impact on the system resources needed to support a
+	   specific DetNet flow will need to be evaluated by the controller plane
+	   based on that flow's traffic specification.  An example traffic 
+	   specification that can be used with MPLS with Traffic Engineering 
+	   (MPLS-TE) can be found in <xref target="RFC2212"/>.
+     </t>
+     <t>
+        DetNet uses flow-specific requirements (e.g., maximum
+        number of out-of-order packets, maximum latency of the flow) for
```
Comment (louberger) [inline mpls/draft-ietf-detnet-mpls.xml:757]: s/uses/implementations can use

File: mpls/draft-ietf-detnet-mpls.xml [hunk 4]
```diff
@@ -732,6 +745,23 @@ DFx = DetNet member flow x over a TE LSP
...
+        configuration of POF-related buffers.  POF implementation details
+		are out-of-scope for this document and POF configuration parameters
+		are implementation specific. The Controller Plane determines and 
+		sets the POF configuration parameters. If PEF is combined with POF,
```
Comment (louberger) [inline mpls/draft-ietf-detnet-mpls.xml:761]: s/determines/is expected toidentify

File: mpls/draft-ietf-detnet-mpls.xml [hunk 5]
```diff
@@ -732,6 +745,23 @@ DFx = DetNet member flow x over a TE LSP
...
+     </t>
+     <t>
+        DetNet uses flow-specific requirements (e.g., maximum
+        number of out-of-order packets, maximum latency of the flow) for
+        configuration of POF-related buffers.  POF implementation details
+		are out-of-scope for this document and POF configuration parameters
+		are implementation specific. The Controller Plane determines and 
+		sets the POF configuration parameters. If PEF is combined with POF, 
+		then their configurations need to be coordinated to ensure the
+		combined functionality.
```
Comment (louberger) [inline mpls/draft-ietf-detnet-mpls.xml:763]: I have no idea what this sentence means.  Can you clarify?

**Pull request merged**